src/qibo/gates/abstract.py

```diff
@@ -276,6 +276,17 @@ def generator_eigenvalue(self):
             f"Generator eigenvalue is not implemented for {self.__class__.__name__}",
         )
 
+    def is_clifford(self) -> bool:
+        """Check if the gate is a Clifford gate."""
+        return False
```

Comment:
This was implemented (as `self.clifford`) in #952. If you don't have it in your branch, try to merge the latest master. If you prefer `self.is_clifford()` instead of `self.clifford` please discuss with @renatomello. I am fine with either approach, but we should only have one to avoid redudancy.